### DISCUSSION TITLE
GLB - Slow loading

### DISCUSSION THREAD

**Discussion by k0mrad** [cables-gl/cables_docs#473] [Answers & Questions] [answered]
Hello, 
I'm trying to understand GLB load times in cables. I have a GLB and textures hosted in s3. I see that the patch loads almost immediately, but the node itself takes at least 5 seconds to show. Has anyone found a solution for this? 

Here is my embed: https://cdpn.io/pen/debug/BaPxwWe/c1250ad61215e9422e04cae451122325
I am testing here as I'm trying to make sure that it is embeddable on our production site.

**Comment by pandrr** [comment]
on dev.cables.gl you can the "view structure" has a debug loading times panel, can you screenshot it ?

**Comment by pandrr** [comment, marked as answer]
found the problem, it is now loading a LOT faster on dev.cables.gl

thanks for reporting!!

**Reply by k0mrad** [reply]
awesome fix. thank you.

**Comment by k0mrad** [comment]
Just wondering if this is going to be pushed to production soon as I still have to use dev to export.
